web/docs/understand/operators/transformations/inspect.md

```diff
@@ -0,0 +1,59 @@
+# inspect
+
+Replaces the input with incremental metrics describing about the input.
```

Comment:
For me, the word "incremental" is kind of misleading. I don't really have a great suggestion right now, but I'd prefer something like "Yields metrics about each received batch of events" and similarly for the other parts of the documentation.